app/models/good.rb

```diff
@@ -1,3 +1,17 @@
 class Good < ApplicationRecord
   belongs_to :basket
+
+  before_save :calulate_tax_update_price
+
+  private
+
+  def calulate_tax_update_price
+    exampt = %w[chocolate book pills]
```

Comment:
In this case, because those values are going to be constant and might be needed elsewhere in the application, a good Ruby practice is to extract it as a constant and place it at the top of the class:

`EXEMPT_FROM_TAX = %w[chocolate book pills]`